src/replicated_log.rs

```diff
@@ -296,3 +319,16 @@ pub enum Event {
     /// 新しい場合には、これとは別に`SnapshotLoaded`イベントが発行される.
     SnapshotInstalled { new_head: LogPosition },
 }
+
+/// `ReplicatedLog` を使うコンポーネントを表すラベル.
+///
+/// ラベルはメトリクスに付与される。
+#[derive(Debug, Clone)]
+pub struct ComponentId(String);
```

Comment:
`raftlog` cannot be distinguished from the view point of clients currently. It's inconvenient for debugging.